cachito/web/models.py

```diff
@@ -930,6 +935,21 @@ def _state_to_json(state):
         rv.update(latest_state)
         return rv
 
+    def _sort_pkgs_in_place(self, packages):
```

Comment:
We have almost the same function for integration tests. `sort_pkgs_and_deps_in_place()`. Maybe we can make them identical. But only if it makes sense for you.

Reply:
Sorry, I see the change in the end, NVM :D